src/components/ViewRequest/Actions/SendRequest.tsx

```diff
@@ -25,7 +25,13 @@ const SendRequest = () => {
   const updateHandler = () => {
     if (request.data && currentStage && currentStage.next !== "") {
       const newStage = currentStage.next;
-      updateStage.mutate({ requestId, newStage });
+      updateStage.mutate({
+        requestId,
+        newStage,
+        ...(currentStage.nextEventTitle && {
+          eventTitle: currentStage.nextEventTitle,
+        }),
+      });
```

Comment:
Why not stick with the coding pattern you already had there?

```suggestion
      const eventTitle = currentStage.nextEventTitle;
      updateStage.mutate({
        requestId,
        newStage,
        eventTitle,
      });
```